plugins/woocommerce/includes/admin/settings/class-wc-settings-emails.php

```diff
@@ -593,6 +650,93 @@ public function email_image_url( $value ) {
 		</tr>
 		<?php
 	}
+
+	/**
+	 * Creates the email font family field with custom font family applied to each option.
+	 *
+	 * @param array $value Field value array.
+	 */
+	public function email_font_family( $value ) {
```

Comment:
Many of the `$value` properties are not used in this method (maybe only `title`, `id`, and `default`?). As I understand it, this code is based on a template, which can accept many of these properties and render it differently, but we have full control over the email font families component. What do you think about simplifying it? Or is there a use case where it's useful to have it there?

Reply:
I was thinking of future extensibility, such as "what if we want to add a tooltip here" etc. Plus, some of these properties are added automatically and not taken from the config array. But I agree that it's better to keep things simple, so I have cleaned it up wherever possible.